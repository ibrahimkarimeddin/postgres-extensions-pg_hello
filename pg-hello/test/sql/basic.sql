CREATE EXTENSION pg_hello;

SET pg_hello.repeat = 2;
SELECT pg_hello('Suleiman') as greet;
SELECT now_ms() > 0 as ok;
SELECT spi_version() like 'PostgreSQL%' as has_ver;
