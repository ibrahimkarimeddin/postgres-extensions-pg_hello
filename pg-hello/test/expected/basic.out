CREATE EXTENSION pg_hello;
SET
           greet            
---------------------------
 Hello, Suleiman! Hello, Suleiman!
(1 row)

  ok  
------
 t
(1 row)

 has_ver 
---------
 t
(1 row)
